contracts/state/State.sol

```diff
@@ -171,15 +156,6 @@ contract State is Ownable2StepUpgradeable, IState {
         return address(verifier);
     }
 
-    /**
```

Comment:
Why have you removed the defaultIdType? I think it still makes sense.

Reply:
how it will be used? see no sence